[rum] Update RUM sdk proxy documentation (#11630)
DataDog/documentation: content/en/real_user_monitoring/faq/proxy_rum_data.md

## SDK initialization

> When you set the `proxyUrl` [initialization parameter][1], all RUM data is sent to the specified url (ex: www.proxy.com/foo).

Reviewer: ```suggestion
When you set the `proxyUrl` [initialization parameter][1], all RUM data is sent to the specified URL (for example, www.proxy.com/foo).
```

## Proxy setup

> When your proxy receives data from the RUM SDK, it must be forwarded to Datadog. The RUM SDK adds the `ddforward` query parameter to all requests to your proxy. This query parameter contains the url where all data must be forwarded to.

Reviewer: ```suggestion
When your proxy receives data from the RUM SDK, it must be forwarded to Datadog. The RUM SDK adds the `ddforward` query parameter to all requests to your proxy. This query parameter contains the URL where all data must be forwarded to.
```

To successfully proxy request to Datadog:

1. Add a `X-Forwarded-For` header containing the request client IP address, for accurate geoIP.

> 2. Forward the request to the url set in the `ddforward` query parameter.

Reviewer: ```suggestion
2. Forward the request to the URL set in the `ddforward` query parameter.
```
